server/game/gamesteps/challenge/fulfillmilitaryclaim.js

```diff
@@ -10,15 +10,24 @@ class FulfillMilitaryClaim extends BaseStep {
     }
 
     continue() {
-        var promptMessage = 'Select ' + this.claim + ' ' + (this.claim > 1 ? 'characters' : 'character') + ' to fulfill military claim';
+        this.forcedClaim = _.filter(this.player.mustChooseAsClaim, card => card.controller === this.player && card.location === 'play area');
+        
+        let claimToSelect = this.claim;
+        
+        if(this.forcedClaim.length < this.claim) {
+            claimToSelect = this.claim - this.forcedClaim.length;
```

Comment:
I'm specifically checking for '<' instead of '<=', so claim to select will always be >0. This does mean that in the most common use case (1 Spearmaiden trigger and 1 claim) the player still gets a normal claim prompt, he'll just have a single option to choose. I could be convinced to skip the prompt whenever this.forcedClaim.length === this.claim, it's basically choosing less clicks versus maintaining clarity of what's going on. <@cryogen 

Another message is useful in any case though.